whisk/client.go

```diff
@@ -616,6 +618,55 @@ func parseApplicationError(resp *http.Response, data []byte, v interface{}) (*ht
 	return resp, whiskErr
 }
 
+func getApplicationErrorMessage(errResp interface{}) string {
```

Comment:
what conditions caused a new format error to be generated? is there an example that can be cited that would be encountered (and added tot he comment). Would love to know where we have seen this error map format.

Reply:
@mrutkows this fix is primarily for usage by the trigger create/delete code.  this cli code invokes the action internally, and needs the error response details.  without this pr, the error details are missing.  here's the cli routine that invokes the client-go code. https://github.com/apache/openwhisk-cli/blob/368da5bf3d34b4a9b64204a0876618d79e014146/commands/trigger.go#L80

without this pr, you can see that the err returned from the `Client.Triggers.Insert` does not have the failure details when you create a trigger using your own "bad" feed action.